### PR TITLE
SIP-91 audit remediation + realtime debt cache external overrides

### PR DESCRIPTION
**Pull request by zyzek**

### CODE REVIEW

**Review by i-stam — APPROVED** [review]
lgtm

File: contracts/DebtCache.sol [hunk 1]
```diff
@@ -90,6 +90,22 @@ contract DebtCache is Owned, MixinResolver, MixinSystemSettings, IDebtCache {
         return getDebtSnapshotStaleTime();
     }
 
+    function cachedDebt() external view returns (uint) {
```
Comment (justinjmoses) [inline contracts/DebtCache.sol:93]: [qu] Any reason to add these given they are auto-added by the compiler?

**Review by justinjmoses — APPROVED** [review]
:shipit:

File: contracts/DebtCache.sol (hunk 1 above)
Comment (zyzek) [reply contracts/DebtCache.sol:93]: Solidity didn't like me overriding the synthesised getters for these public members the public members in `RealtimeDebtCache` unless I explicitly exposed them this way.

**Pull request merged**